packages/kbn-text-based-editor/src/query_history.tsx

```diff
@@ -103,30 +123,60 @@ export const getTableColumns = (
           case 'success':
           default:
             return (
-              <EuiIcon
-                type="checkInCircleFilled"
-                color="success"
-                size="s"
-                data-test-subj="TextBasedLangEditor-queryHistory-success"
-              />
+              <EuiToolTip
+                position="top"
+                content={i18n.translate(
+                  'textBasedEditor.query.textBasedLanguagesEditor.querieshistory.success',
+                  {
+                    defaultMessage: 'Query run successfully',
```

Comment:
```suggestion
                    defaultMessage: 'Query ran successfully',
```

What you have isn't wrong, but FWIW I like matching the verb tense with the other message "query failed"